node/external/transactionAPI/gasUsedAndFeeProcessor.go

```diff
@@ -58,24 +58,17 @@ func (gfp *gasUsedAndFeeProcessor) computeAndAttachGasUsedAndFee(tx *transaction
 	}
```

Comment:
line 45 to 57 is almost the same with 
https://github.com/multiversx/mx-chain-go/pull/6546/files#diff-ed16c2d2641c2e82d0ae0481dc9722266c5f31e111d8fac23ed87f9598e2ae85R131